docs/GCE.md

```diff
@@ -1,63 +1,110 @@
+```
+curl https://discovery.etcd.io/new?size=3
+```
+You need to grab a fresh URL each time you bootstrap a cluster.
 
-Run the master with
+Create a file `cloud-config.yaml` with the following contents; **replace `<discovery URL>` with the URL retrieved above**:
 ```
-gcloud compute instances create master --image https://www.googleapis.com/compute/v1/projects/coreos-cloud/global/images/coreos-alpha-618-0-0-v20150312 --machine-type g1-small --metadata-from-file user-data=cloud-config-master.yaml --can-ip-forward
+#cloud-config
+coreos:
+  update:
+    reboot-strategy: etcd-lock
+  etcd2:
+    name: $private_ipv4
+    discovery: <discovery URL>
+    advertise-client-urls: http://$private_ipv4:2379
+    initial-advertise-peer-urls: http://$private_ipv4:2380
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://$private_ipv4:2380,http://$private_ipv4:7001
+  units:
+    - name: etcd2.service
+      command: start
+
 ```
+Note: we disable automated reboots for this demo to avoid interrupting the instructions.
 
-And the compute nodes with
+Then create the cluster with the following command ("calico-1 calico-2 calico-3" is the list of nodes to create):
 ```
-gcloud compute instances create core1 core2 --image https://www.googleapis.com/compute/v1/projects/coreos-cloud/global/images/coreos-alpha-618-0-0-v20150312 --machine-type g1-small --metadata-from-file user-data=cloud-config.yaml --can-ip-forward
+gcloud compute instances create \
+  calico-1 calico-2 calico-3 \
+  --image https://www.googleapis.com/compute/v1/projects/coreos-cloud/global/images/coreos-alpha-709-0-0-v20150611 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config.yaml
 ```
 
-You can get the cloud-config*.yaml files from the tests/scale directory of this repo.
+## Installing calicoctl on each node
+On each node, run these commands to set up Calico:
+```
+# Download calicoctl and make it executable:
+wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.5/calicoctl
+chmod +x ./calicoctl
 
-## Setting up GCE networking
-In order for routing to work correctly between hosts, you must set up a firewall rule on GCE to allow IPIP packets to flow between the instances. The command for doing this is as follows.
+# Grab our private IP from the metadata service:
+export metadata_url="http://metadata.google.internal/computeMetadata/v1/"
+export private_ip=$(curl "$metadata_url/instance/network-interfaces/0/ip" -H "Metadata-Flavor: Google")
 
+# Start the calico node service:
+sudo ./calicoctl node --ip=$private_ip
+
+# Work-around a [BIRD routing issue](http://marc.info/?l=bird-users&m=139809577125938&w=2):
```

Comment:
I would be temped to say:
# Add a route to point BIRD in the right direction (as suggested [here])